ansys/mapdl/core/examples/examples.py

```diff
@@ -11,6 +11,3 @@
 # This way, files can be loaded with:
 # from ansys.mapdl.core import examples
 # examples.wing_model
-
-# downloadable files
-from .downloads import download_bracket
```

Comment:
I think we still need this here since we `from .examples import *`

If you can think of a better way...

Reply:
In `examples/__init__.py` there is `from .downloads import *`. Won't this cover the use of download_bracket?

Reply:
Good point.  Thanks!